src/langchain_google_cloud_sql_mysql/mysql_engine.py

```diff
@@ -164,3 +164,55 @@ def connect(self) -> sqlalchemy.engine.Connection:
                 out from the connection pool.
         """
         return self.engine.connect()
+
+    def init_document_table(
+        self,
+        table_name: str,
+        metadata_columns: List[sqlalchemy.Column] = [],
+        store_metadata: bool = True,
+    ) -> None:
+        """
+        Create a table for saving of langchain documents.
+
+        Args:
+            table_name (str): The MySQL database table name.
+            metadata_columns (List[sqlalchemy.Column]): A list of SQLAlchemy Columns
+                to create for custom metadata. Optional.
+            store_metadata (bool): Whether to store extra metadata in a metadata column
+                if not described in 'metadata' field list (Default: True).
+        """
+        columns = [
+            sqlalchemy.Column(
+                "page_content",
+                sqlalchemy.UnicodeText,
+                primary_key=False,
+                nullable=False,
+            )
+        ]
+        columns += metadata_columns
+        if store_metadata:
+            columns.append(
+                sqlalchemy.Column(
+                    "langchain_metadata",
+                    sqlalchemy.JSON,
+                    primary_key=False,
+                    nullable=True,
+                )
+            )
+        metadata = sqlalchemy.MetaData()
+        sqlalchemy.Table(table_name, metadata, *columns)
+        metadata.create_all(self.engine)
```

Comment:
@loeng2023 probably better to use `create` instead of `create_all` here since we are only creating a single table.

Ref: https://docs.sqlalchemy.org/en/20/core/metadata.html#creating-and-dropping-database-tables
```suggestion
        doc = sqlalchemy.Table(table_name, metadata, *columns)
        doc.create(self.engine)
```

Reply:
Thanks for suggestion. Updated to use `create`.